timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -92,13 +92,12 @@ class TimeSeriesPredictor:
         If True, the predictor will ignore the datetime indexes during both training and testing, and will replace
         the data indexes with dummy timestamps in second frequency. In this case, the forecast output time indexes will
         be arbitrary values, and seasonality will be turned off for local models.
-    learner_type : AbstractLearner, default = TimeSeriesLearner
-        A class which inherits from ``AbstractLearner``. The learner specifies the inner logic of the
-        ``TimeSeriesPredictor``.
-    learner_kwargs : dict, optional
-        Keyword arguments to send to the learner (for advanced users only). Options include ``trainer_type``, a
-        class inheriting from ``AbstractTrainer`` which controls training of multiple models.
-        If ``path`` and ``eval_metric`` are re-specified within ``learner_kwargs``, these are ignored.
```

Comment:
We provide no documentation explaining how these kwargs could be used, so I removed them from the API to reduce clutter.